dZoneManager/dZoneManager.h

```diff
@@ -58,7 +61,10 @@ class dZoneManager {
 
 	Zone* m_pZone = nullptr;
 	LWOZONEID m_ZoneID;
-	bool m_PlayerLoseCoinsOnDeath; //Do players drop coins in this zone when smashed
+	bool m_PlayerLoseCoinsOnDeath;
+	bool m_DisableSaveLocation;
+	bool m_MountsAllowed;
+	bool m_PetsAllowed;
```

Comment:
These should be default initializing to something, not sure if false or true is correct for default however.